Doc(release) Release notes collect 23.10.15 (#4282)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx

## Centreon Collect

### 23.10.15

Release date: `April 8, 2025`

<details>
  <summary>Bug fixes</summary>

> - [Broker] Fixed an issue with acknowledgements that did not appear in the logs table.

Reviewer: ```suggestion
- [Broker] Fixed an issue where acknowledgements did not appear in the logs table.
```

> - [Configuration] Fixed issue with the output password parameter of the Broker stream connector.

Reviewer: ```suggestion
- [Configuration] Fixed an issue where password parameters were incorrectly saved when creating several output-type stream connectors.
```
